y_pef_schedule.install

```diff
@@ -73,7 +39,7 @@ function _y_pef_schedule_update_displays_on_install(): void {
   // Retrieve and update the form display for 'activity' nodes.
   $form_display = $entity_display_repository->getFormDisplay('node', 'activity');
   $form_display->setComponent('field_activity_color', [
-    'type' => 'string_textfield',
+    'type' => 'colorapi_color_widget',
```

Comment:
we need to ensure upgrade path is supported
What happens with existing installation?
Are we sure nobody uses this to date?

Reply:
we are in rc phase, so I assume we are safe, but we would need to describe after rc6 we have broken upgrade path ( possibly )